src/main.cpp

```diff
@@ -173,6 +180,9 @@ static void print_usage()
             "\t -t <timeout>              Timeout (in seconds)\n"
             "\t -o <format>               Output format (default, csv, kv, or json)\n"
             "\t -x <plugin>               Don't activate the specified plugin\n"
+            "\t --stop <plugin>           Stop the specified plugin before termination loop\n"
+            "\t --wait-stop-plugins <timeout>\n"
```

Comment:
I don't think these options should be needed. Plugins by default should be terminated safely - which is what this stop-plugin infrastructure accomplishes. So if right now plugins can be terminated by pressing ctrl+c such that it leaves things unstable in the VM is bad. The default operation should be to always exit such that the VM is left in a stable state.

Reply:
The only option we should have is just the time to wait for plugins to stop when given the signal. The default should be a safe 10 seconds, but if the user specifies 0 no stop-wait-loop happens.

Reply:
OK